server/controllers/dishes/deleteDish.ts

```diff
@@ -0,0 +1,25 @@
+import { Request, Response, NextFunction } from 'express';
+import { Dish } from '../../models';
+import { DishesAttributes } from '../../utils/types';
+import { StatusCodes } from '../../utils/enum';
+import { CustomError } from '../../utils';
+
+const deleteDish = async (req: Request, res: Response, next: NextFunction) => {
+  const { id } = req.params;
```

Comment:
Destructure the id in the try statement's scoop. Think what will happen when the user doesn't send an id in the params.

Reply:
thank you for this note ❤️
done 👍